controller/aks-cluster-config-handler.go

```diff
@@ -283,6 +283,12 @@ func (h *Handler) checkAndUpdate(config *aksv1.AKSClusterConfig) (*aksv1.AKSClus
 		return config, err
 	}
 
+	if config.Status.RBACEnabled == nil && result.EnableRBAC != nil {
+		config = config.DeepCopy()
+		config.Status.RBACEnabled = result.EnableRBAC
+		return h.aksCC.UpdateStatus(config)
```

Comment:
>  In order to warn a user....

Meaning that this will just exist on status and the UI will be the one displaying the error message ? We'll need to make a UI issue (if Wes isn't already working on this one).